api/server/router/volume/volume_routes.go

```diff
@@ -517,6 +523,10 @@ func (r *router) volumeAttach(
 			}
 		}
 
+		if v.AttachmentState == 0 {
+			v.AttachmentState = types.VolumeAttached
+		}
+
```

Comment:
why is this one `0 -> attached` and all the others `0 -> available`? My guess is that this is basically setting the default value, and if a call to `Attach()` is returning without an error, it can be assumed that the volume is attached.  makes sense, but want to double check.